app/controllers/events/view/speakers/list.js

```diff
@@ -15,7 +15,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name            : 'Name',
         valuePath       : 'name',
-        extraValuePaths : ['id'],
+        extraValuePaths : ['id', 'event'],
```

Comment:
```suggestion
        extraValuePaths : ['id'],
```

Reply:
We need to provide event in it, without this we won't be able to get event id bcz this.extraRecords is not complete speaker object just properties of speaker that are defined in this extraValuePaths.